src/schemas/v1/incentive.ts

```diff
@@ -74,6 +129,8 @@ export const API_INCENTIVE_SCHEMA = {
     },
     owner_status: {
       type: 'array',
+      description: `Whether the consumer must be a homeowner or renter (or \
+either) to claim this incentive.`,
```

Comment:
Do we present homeowner or renter as either in the api or is it returned as `both?

Reply:
We use multiple values:

```
    "owner_status": [
      "homeowner",
      "renter"
    ],
```